### ISSUE TITLE
[On Addon Support] Is it feasible to be able to add any of the currently recommended addons?

### ISSUE THREAD

**Issue by fabiorzfreitas** [allo-/ffprofile#289]
As of today, Mozilla has 98 addons badged as [recommended](https://addons.mozilla.org/en-US/firefox/search/?promoted=recommended&type=extension).

I'm not sure how addon support is implemented in this project, but perhaps there could be a `More...` button on the UI to pop up a window with checkboxes for those addons, without the need to bloat the current page!

**Comment by allo-** [comment]
Currently either a enterprise policy is generated for installing the latest version or an outdated version is put into the zip file, that you should update right after download.

Adding a broader range of addons (without recommending them) would be a idea, but I think it should be outside of the other profile creation, possibly using a more compact format (checkbox, title, short description, link) as well.

On the other hand, I am not sure if one cannot simply use the Mozilla page. Or maybe it could be linked at the end somewhere.

**Comment by fabiorzfreitas** [comment]
On Profiles and #71, I didn't notice that `02_private_browsing.json` is not a subset of `01_default.json`. Perhaps the dropdown menu could offer 2 more profiles: `03_both.json`, which would contain every setting from the 2 others (maybe even something like `"Extended Privacy": [...]` to this profile), and `4_all_plus_addons`.

This last profile would inherit every setting from `03_both.json`, but it would feature all the 98 addons mentioned. In a beta version, the UI would be the same and every addon would have its `addon.json` file created manually, just like the current ones.

I would guess this Recommended badge list doesn't change that often, so constantly comparing with the Mozilla page wouldn't be strictly required and that any change could be reported by users.

If I have the time to do so, I'll try to create a fork for this :)

**Comment by allo-** [comment]
Yes, the private browing profile is more a proof of concept and possibly should be removed. I had to rename it to "private browsing only" as it already caused confusion.

Ideally there would be a large pool of settings, a nice set of profiles grouping them for different purposes and then presets (#57) for these profiles like "paranoid" / "normal" / "convenient" that change the default values.
